Tutorial needs a note to check out v1.0 of ecs

Hi,

I'm working through the tutorial and I found that when I got to the end of 01-hello-world, I was encountering the following error:

```
$ go run traffic.go
# engo.io/engo/common
../../../engo.io/engo/common/animation.go:99:11: invalid map key type ecs.BasicEntity
../../../engo.io/engo/common/animation.go:110:21: invalid map key type ecs.BasicEntity
```

After a bit of thinking and fiddling, I ran:

```
$ cd /src/engo.io/ecs/
$ git checkout v1.0
```

Changing back to the TrafficManager directory and running `go run` worked.

(Apologies for not raising a PR to fix the doc but I wasn't sure about the best place to put a note about this. In the foreword?)

--- a/tutorials/_posts/2016-04-08-foreword.md
+++ b/tutorials/_posts/2016-04-08-foreword.md
@@ -68,7 +68,8 @@ up a terminal / command prompt, go to this location (using the `cd` command), an
 
 This tutorial series is built based on engo v1.0, and the master branch possibly contains API changes or other
 development changes that may not work exactly as stated in the tutorial. To ensure you're using v1.0, do
-`cd $GOPATH/src/engo.io/engo` and then checkout the right version with `git checkout v1.0`.
+`cd $GOPATH/src/engo.io/engo` and then checkout the right version with `git checkout v1.0`. This also depends
+on using v1.0 of ecs, so also do `cd $GOPATH/src/engo.io/ecs` and then checkout the right version with `git checkout v1.0`.
 
 This command will fetch `engo` from the repository, including all its dependencies.
 

